features/upgrade/sdn/egress-upgrade.feature

```diff
@@ -261,6 +263,7 @@ Feature: Egress compoment upgrade testing
   @proxy @noproxy @disconnected @connected
   @arm64 @amd64
   Scenario: Check sdn egressip is functional post upgrade
+    Given the env is using "OpenShiftSDN" networkType
```

Comment:
Should post upgrade checking be "Given the cluster is not migration from sdn plugin"?

Reply:
yes, you are right. I will add the step.
 I thought we can check this feature after sdn-migration-rollback.  but seems this resource will be deleted after migration to OVN.  So even roll back to openshift-sdn. the resource will not be back.

Reply:
added this step for unsupported features including egressip/egressfirewall/multicast. etc.

Reply:
> yes, you are right. I will add the step. I thought we can check this feature after sdn-migration-rollback. but seems this resource will be deleted after migration to OVN. So even roll back to openshift-sdn. the resource will not be back.

"Given the cluster is not migration from sdn plugin" is not used for "post upgrade - prepare".
Should line 267 be replaced by "Given the cluster is not migration from sdn plugin" and remove "Given the cluster is not migration from sdn plugin" line in 193?

Reply:
yes..good catch. updated. thanks.